modules/OFConnectionManager2/module/src/cxn_instance.c

```diff
@@ -81,6 +81,16 @@ cxn_try_to_connect(connection_t *cxn);
 /* Maximum number of messages to send per write callback */
 #define MAX_WRITE_MSGS 32
 
+/* Former cxn outstanding async count
```

Comment:
sorry, i don't like the "former" naming.  to me, the "former async pending count" means "we used to have some pending async operations, but now they aren't pending".  can we rename this?  maybe "previous_pending_cnt"?  "accumulated_pending_cnt"?  "pending_cnt_excluding_current"?

Reply:
sure